error.go

```diff
@@ -21,9 +21,8 @@
 package zap
 
 import (
-	"sync"
-
 	"go.uber.org/zap/zapcore"
+	"sync"
 )
```

Comment:
this doesn't need to change

```suggestion
	"sync"

	"go.uber.org/zap/zapcore"
)
```